extensions/cornerstone/src/services/ViewportActionCornersService/ViewportActionCornersService.ts

```diff
@@ -51,28 +49,115 @@ class ViewportActionCornersService extends PubSubService {
     if (addComponentsImplementation) {
       this.serviceImplementation._addComponents = addComponentsImplementation;
     }
-    if (clearComponentsImplementation) {
-      this.serviceImplementation._clear = clearComponentsImplementation;
+    if (clearImplementation) {
+      this.serviceImplementation._clear = clearImplementation;
```

Comment:
If you do the change I suggest below, then you can write these without if, as:

this.clear = clearImplementation ?? this.clear;